holopy/inference/tests/test_interface.py

```diff
@@ -185,14 +184,45 @@ def test_make_default_model_construction(self):
 
     @attr('fast')
     def test_parameterize_scatterer_makes_priors(self):
-        scatterer = parameterize_scatterer(Sphere(), 'r')
+        scatterer = parameterize_scatterer(Sphere(), ['r'])
         self.assertTrue(isinstance(scatterer.r, prior.Prior))
 
     @attr('fast')
-    def test_rename_xyz(self):
-        parameters_list = ['x', 'p', 'y']
-        parameters_list = rename_xyz(parameters_list)
-        self.assertEqual(parameters_list, ['center.0', 'p', 'center.1'])
+    def test_parameterize_scatterer_xyz(self):
+        scatterer = parameterize_scatterer(Sphere(center=[0, 0, 0]), ['x'])
+        self.assertTrue(isinstance(scatterer.center[0], prior.Prior))
+
+    @attr('fast')
+    def test_parameterize_scatterer_center(self):
+        scatterer = parameterize_scatterer(Sphere(center=[0, 0, 0]), ['center'])
+        expected = prior.Uniform(-np.inf, np.inf, 0, 'x')
+        self.assertEqual(scatterer.center[0], expected)
```

Comment:
Does `parameterize_scatterer(Sphere(center=[0, 0, 0]), ['center'])` return all 3 priors for x, y, and z, or just x? If so, we could do
```python
for i, xyz in enumerate('xyz'):
    expected = prior.Uniform(-np.inf, np.inf, 0, xyz)
    self.assertEqual(scatterer.center[i], expected)
```